templates/field/field--node--oe-departments--oe-person--teaser.html.twig

```diff
@@ -6,4 +6,4 @@
  * @see ./core/themes/stable/templates/field/field.html.twig
  */
 #}
-{%- include 'field--multiple-values--pipe-separated.html.twig' -%}
+{%- include 'field--multiple-values--comma-separated.html.twig' -%}
```

Comment:
This change has to be done for all the departments fields according to the ticket.